_posts/2016-03-28-carbon-route-released.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "carbon-route element beta release"
+date:   2016-03-28 10:30:00
+categories: announcements
+author: polymer-team
+excerpt: "The `carbon-route` element has made its beta debut - we'd love your feedback on this modular approach to routing."
+
+---
+
+We're excited to announce the beta release of the [`carbon-route`](https://github.com/polymerelements/carbon-route) element - a composable, modular web component router! To dive into this new element and see what it can do, check out the [Encapsulated Routing with Elements](https://www.polymer-project.org/1.0/articles/routing.html) article on the Polymer site.
+
+The `carbon-route` element provides a declarative way to let web components hook in to the current URL, and route behavior accordingly. It is currently in its `v0.8` iteration - complete, but still may evolve based on feedback before it reaches `v1.0`. As an example of how the `carbon-route` element may be used:
+
+```
+<carbon-route pattern="/tabs/:tabName" data="{{data}}"></carbon-route>
```

Comment:
Thanks @abdonrd! Completeness will be good, I'll make sure to include `carbon-location` as well.